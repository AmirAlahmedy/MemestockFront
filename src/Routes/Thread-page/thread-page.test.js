// import React from 'react';
// import threadPage from './thread-page';
// import ReactDOM from 'react-dom';


// it('A listning renders without crashing', () => {
//     const div = document.createElement('div');
//     ReactDOM.render(<threadPage />, div);
//     ReactDOM.unmountComponentAtNode(div);
//   });
