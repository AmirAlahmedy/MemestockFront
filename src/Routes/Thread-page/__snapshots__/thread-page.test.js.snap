// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subreddit component snapshot it matches the snapshot 1`] = `
<div
  className="page"
>
  <div
    class="threadPageContainer"
  >
    <div
      className="PageThread"
    >
      <div>
        <div
          className="classicThreadContainer"
        >
          <div
            class="threadLinks"
          >
            <br />
            <span
              className="threadSubreddit"
              onClick={[Function]}
            >
               r/
              http://localhost/
            </span>
            .
            <span
              className="posted-by"
              onClick={[Function]}
            >
                 Posted by u/
              
               
            </span>
          </div>
          <br />
          <div
            className="threadTitle"
          >
            
          </div>
          <span
            className="threadComments"
            onClick={[Function]}
          >
            View
          </span>
          <p
            className="classicThreadContent"
          >
            
          </p>
          <button
            className="incrementVotes"
            onClick={[Function]}
            type="button"
          >
            <i
              class="fas fa-angle-up"
            />
          </button>
          <div
            className="threadUpvotes"
          >
            0
          </div>
          <button
            className="decrementVotes"
            onClick={[Function]}
            type="button"
          >
            <i
              class="fas fa-angle-down"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      class="addCommentSection"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          className="textComment"
          name="comment"
          onChange={[Function]}
          placeholder="Write your comment here..."
          type="text"
          value=""
        />
        <input
          className="saveComment"
          type="submit"
          value="Save"
        />
      </form>
      <label
        className="lockComment"
        for="checkLocked"
      >
        Lock
      </label>
      <input
        id="checkLocked"
        name="checkLocked"
        type="checkbox"
      />
      <label
        className="spoilerComment"
        for="checkSpoiler"
      >
        Mark as spoiler
      </label>
      <input
        id="checkSpoiler"
        name="checkSpoiler"
        type="checkbox"
      />
    </div>
    <ul
      class="commentList"
    />
    <div />
    <div />
  </div>
  <div
    className="threadPageSidebarContainer"
  >
    <div />
    <div />
  </div>
</div>
`;
